pkg/controllers/provisioning/provisioner.go

```diff
@@ -157,6 +157,7 @@ func (p *Provisioner) launch(ctx context.Context, constraints *v1alpha5.Constrai
 		pods <- ps
 	}
 	return p.cloudProvider.Create(ctx, constraints, packing.InstanceTypeOptions, packing.NodeQuantity, func(node *v1.Node) error {
+		// Inserting labels from constraints
```

Comment:
I'm not sure this comment adds much value. 
```suggestion
```

Reply:
done